packages/opentelemetry-exporter/.ncurc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "reject": ["mocha", "eslint-plugin-mocha"]
+}
```

Comment:
What is ncurc.json? :)

Reply:
https://github.com/raineorshine/npm-check-updates#configuration-files

See also https://github.com/instana/nodejs/blob/main/package.json#L31